src/components/button-bar/button-bar.spec.tsx

```diff
@@ -278,4 +279,30 @@ describe("with different button props", () => {
     );
     expect(wrapper.containsMatchingElement(<span>subtext</span>)).toBeTruthy();
   });
+
+  describe("Button Minor variant", () => {
```

Comment:
I think something like `with ButtonMinor children` would be a little clearer here as it's not a `variant`